### ISSUE TITLE
[Accessibility]: Add shortcut keys to jump between executed commands in terminal buffer

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#179272]
CC @meganrogge

NVDA has a really great example model on this feature. I recommend experiencing this:

1. Load NVDA.

1. Press Ctrl+NVDA+Z to open NVDA's built-in Python repl.

1. Type and run multiple Python commands, such as `print("hello"); print("world");` (execute separately).
1. Press F6 to focus in the output buffer where you can use standard arrows and navigation keys to review. This is pretty much the same as VSCode a11y terminal buffer.

1. In the output buffer, press Alt+UpArrow and Alt+DownArrow to quickly jup between executed commands.

* Note: Pressing Alt+Up/DownArrows let you move through your command history. This is somewhat similar to VSCode `Ctrl+Shift+O` behavior, but this is different b/c users do not need to open additional view.

It would be so great if Alt+Up/DownArrow keys can be used in VSCode a11y terminal buffer to implement this feature.

**Comment by meganrogge** [comment]
Our implementation of `Ctrl+Shift+O` relies on shell integration to know where the commands are. 

Are the types of commands you run in the terminal within a python subshell typically? If so, we wouldn't be able to support this. If not, we could consider adding this.

**Comment by jooyoungseo** [comment]
@meganrogge  -- Although a general solution would be ideal, implementing this in integrated shell might be a great start and it will be instrumental a lot for sure.

**Comment by Tyriar** [comment]
@meganrogge let's use ctrl/cmd+up/down as this is essentially just asking for the regular command navigation but in the accessible buffer?

**Comment by meganrogge** [comment]
Yep that was my thought too

**Comment by meganrogge** [comment]
To verify, in a terminal with shell integration, run some commands. 

Enter the accessible buffer with `Shift+tab`. `Ctrl/cmd+Up/DownArrow` should go between the commands

**Comment by roblourens** [comment]
That works, but I also see this

- With screen reader mode enabled
  - I can enter the accessible buffer and can use cmd+up
  - I can't use cmd+up to scroll to the previous command when _not_ in the accessible buffer
  - I can use cmd+shift+up to select to the previous command
- Without screen reader mode enabled
  - I can't enter the accessible buffer, seems that it just sends tab to my shell
  - I can use cmd+up to scroll to the previous command

I see that the "scroll to previous command" keybinding has this when clause `terminalFocus && terminalHasBeenCreated && !accessibilityModeEnabled || terminalFocus && terminalProcessSupported && !accessibilityModeEnabled` but the "select to previous command" doesn't have the `!accessibilityModeEnabled` checks, is that expected?
